authors/apps/articles/test/test_create_article.py

```diff
@@ -0,0 +1,130 @@
+from authors.apps.articles.models import Article
+# from authors.apps.authentication.views import LoginAPIView, RegistrationAPIView
+
+class CreateArticles(APITestCase):
+    def setUp(self):
+        self.client = APIClient()
+        self.login_url = reverse('authentication:auth-login')
+        self.signup_url = reverse('authentication:auth-register')
+        self.listcreate = reverse('articles:articles-listcreate')
+        self.signup_data = {
+            "user": {
+                "username": "kennyg",
+                "email": "ken@me.com",
+                "password": "Kennyisme1!"
+                }}
+        self.login_data = {
+            "user": {
+                "email": "ken@me.com",
+                "password": "Kennyisme1!"
+                }}
+        self.signup_data2 = {
+            "user": {
+                "username": "kwammea",
+                "email": "kwame@me.com",
+                "password": "Kwameisme1!"
+                }}
+        self.create_article_data = {
+            "title": "A new story",
+            "body": "This is my story",
+            "description": "Here is my story",
+            "images": None
+        }
+        self.create_article_data2 = {
+            "title": "The beginning",
+            "body": "This begins nowy",
+            "description": "Here is my story",
+            "images": None
+        }
+        self.delete_article = {
+            "is_displayed": False
+        }
+
+    def test_fetch_all_articles(self):
+        """Tests to fetch all articles"""
+        register = self.client.post(self.signup_url,
+                                    self.signup_data,
+                                    format='json')
+        token = register.data['token']
+        del self.create_article_data['images']
+        response = self.client.post(self.listcreate,
+                                    self.create_article_data, format='json',
+                                    HTTP_AUTHORIZATION='token {}'.format(token))
+        response = self.client.post(self.listcreate,
+                                    self.create_article_data, format='json',
+                                    HTTP_AUTHORIZATION='token {}'.format(token))
+        response = self.client.get(self.listcreate, format='json')
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+    
+    def test_add_articles(self):
+        """Test to add an article"""
+        register = self.client.post(self.signup_url,
+                                    self.signup_data,
+                                    format='json')
+        token = register.data['token']
+        del self.create_article_data['images']
+        response = self.client.post(self.listcreate,
+                                    self.create_article_data, format='json',
+                                    HTTP_AUTHORIZATION='token {}'.format(token))
+        self.assertEqual(register.status_code, status.HTTP_201_CREATED)
```

Comment:
> Then I think you are supposed to assert response instead of register.

The response is being saved as a variable called `register`. Renaming it would perhaps avoid confusion